mesos-docker/run/cfg.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+type jq >/dev/null 2>&1 || { echo >&2 "jq binary is required but it's not installed (https://stedolan.github.io/jq/).  Aborting."; exit 1; }
+
+function get_number_of_slaves {
+
+  jq '.configs |  length' $1
+
+}
+
+
+function get_slave_cfg_from_index {
+
+  RET=$(jq ".configs[$2]" $1)
+
+  if [[ "$RET" == null ]] || [[ "$RET" == "\"\"" ]]; then
+    echo ""
+  else
+    echo "$RET"
+  fi
+
+}
+
+
+function get_field_value_from_slave_cfg {
+
+  RET=$(jq ".$1" $2)
+
+  if [[ "$RET" == null ]] || [[ "$RET" == "\"\"" ]] ; then
+    echo ""
+  else
+    echo "$RET"
+  fi
+
+}
+
+function get_field_value_from_slave_cfg_at_index {
```

Comment:
It would be good to have some documentation on these functions.
Something like:

```
<description of what the function does>
$1 - <what is used for>
$2 - <this guy too>
$3 - <another one !!>
```